Mathlib/MeasureTheory/Function/SimpleFunc.lean

```diff
@@ -1312,6 +1312,33 @@ protected theorem induction {α γ} [MeasurableSpace α] [AddMonoid γ] {P : Sim
     · simp [piecewise_eq_of_not_mem _ _ _ hy, -piecewise_eq_indicator]
 #align measure_theory.simple_func.induction MeasureTheory.SimpleFunc.induction
 
+/-- In a topological vector space, the addition of a measurable function and a simple function is
+measurable. -/
+theorem _root_.Measurable.add_simpleFunc
```

Comment:
Do you think it's worth having `simpleFunc_add` too for convenience?

Reply:
Added. I copied the proof because the group is not assumed to be commutative...